examples/swarm/test_db.py

```diff
@@ -103,12 +103,13 @@ def dbValidator():
   print "    host   :    ", host
   print "    port   :    ", port
   print "    user   :    ", user
-  print "    passwd :    ", passwd
+  print "    passwd :    ", "*" * len(passwd)
 
 
-  if testDbConnection(host, port, user, passwd):
+  try: 
+    testDbConnection(host, port, user, passwd)
     print "Connection successful!!"
-  else:
+  except pymysql.err.OperationalError:
```

Comment:
If you don't re-raise the exception after printing the text, then higher level program that invokes this script has no way to find out that something went wrong. Actually, just remove the try/except here and move printing of the test to the `pymysql.err.OperationalError` handler in `testDbConnection` (instead of `print "Error: failed to connect to database"` there).